.evergreen/run-datalake-tests.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
```

Comment:
It seems like this script is mostly copy pasted from run-tests.sh. Can we merge `test.data_lake.test_data_lake` into run-tests.sh instead of adding a new script?